yt_dlp/extractor/khanacademy.py

```diff
@@ -62,11 +68,13 @@ def _parse_component_props(self, component_props):
         video = component_props['tutorialPageData']['contentModel']
         info = self._parse_video(video)
         author_names = video.get('authorNames')
-        info.update({
-            'uploader': ', '.join(author_names) if author_names else None,
-            'timestamp': parse_iso8601(video.get('dateAdded')),
-            'license': video.get('kaUserLicense'),
-        })
+        info.update(
+            {
+                'uploader': ', '.join(author_names) if author_names else None,
+                'timestamp': parse_iso8601(video.get('dateAdded')),
+                'license': video.get('kaUserLicense'),
+            }
+        )
```

Comment:
unnecessary change

Reply:
Got it. I will turn off black auto-formatting.